cpp/CMakeLists.txt

```diff
@@ -24,7 +24,11 @@ include(rapids-cuda)
 include(rapids-export)
 include(rapids-find)
 
-rapids_cuda_init_architectures(RAPIDS_TRITON)
+option(TRITON_ENABLE_GPU "Enable GPU support in Triton" ON)
+
+if(TRITON_ENABLE_GPU)
+  rapids_cuda_init_architectures(RAPIDS_TRITON)
+endif()
```

Comment:
Not a big deal by any stretch, but why separate this option with the cuda_init before the other options? Was thinking it might look a bit cleaner to have all options before cuda_init and project

Reply:
Yep, I think that makes sense. I did it that way to mirror other RAPIDS-CMake projects as closely as possible. but I do think it's cleaner all together. I'll make that happen.